var/spack/repos/builtin/packages/lua/package.py

```diff
@@ -54,7 +54,7 @@ def install(self, spec, prefix):
              'MYLDFLAGS=-L%s -L%s' % (
                  spec['readline'].prefix.lib,
                  spec['ncurses'].prefix.lib),
-             'MYLIBS=-lncursesw -ltinfow',
+             'MYLIBS=-lncurses -ltinfo',
```

Comment:
I think this would be better as:
```suggestion
             'MYLIBS=' + spec['ncurses'].libs.link_flags,
```
That might even allow you to relax the dependency on `ncurses` so you can build `lua ^ncurses~termlib`